src/RobotInterface/YarpImplementation/include/BipedalLocomotion/RobotInterface/YarpSensorBridge.h

```diff
@@ -52,11 +52,16 @@ namespace RobotInterface
  * |     Group                  |         Parameter               | Type              |                   Description                   |
  * |:--------------------------:|:-------------------------------:|:-----------------:|:---------------------------------------------- :|
  * |                            |check_for_nan                    | boolean           |flag to activate checking for NANs in the incoming measurement buffers, not applicable for images|
- * |                            |stream_joint_states              | boolean           |Flag to activate the attachment to IMU sensor devices       |
+ * |                            |stream_joint_states              | boolean           |Flag to activate the attachment to remapped control boards for joint states reading     |
  * |                            |stream_inertials                 | boolean           |Flag to activate the attachment to IMU sensor devices       |
  * |                            |stream_cartesian_wrenches        | boolean           |Flag to activate the attachment to Cartesian wrench related devices       |
  * |                            |stream_forcetorque_sensors       | boolean           |Flag to activate the attachment to six axis FT sensor devices       |
  * |                            |stream_cameras                   | boolean           |Flag to activate the attachment to Cameras devices       |
+ * |                            |stream_pids                      | boolean           |Flag to activate the attachment to remapped control boards for pids reading      |
+ * |                            |stream_motor_states              | boolean           |Flag to activate the attachment to remapped control boards for motor states reading      |
+ * |                            |stream_motor_PWM                 | boolean           |Flag to activate the attachment to remapped control boards for PWM reading      |
+ * |                            |stream_joint_torques             | boolean           |Flag to activate the attachment to remapped control boards for joint torques reading      |
```

Comment:
Why this one is not inside `stream_joint_states`

Reply:
I thought of keeping it separate because the device used to read the measurements is different.

Reply:
I'm not super convinced about this choice. Indeed even if the interfaces are different the joint torque is _de facto_ a joint state. We do not have `steram_joint_position` or `stream_joint_velocity`

Reply:
Ok, you convinced me. I'll change it.